app/javascript/mastodon/locales/ar.json

```diff
@@ -142,6 +142,7 @@
   "status.load_more": "حمّل المزيد",
   "status.media_hidden": "الصورة مستترة",
   "status.mention": "أذكُر @{name}",
+  "status.mute_conversation": "Disable notifications",
```

Comment:
Will all these `locales/*.json` (other than `en`) not just fallback to `en` if they don't exist?